test/integration/services/data/test-get-number-of-submitted-claims-for-date-range.js

```diff
@@ -0,0 +1,51 @@
+const expect = require('chai').expect
+const config = require('../../../../knexfile').asyncworker
+const knex = require('knex')(config)
+const testHelper = require('../../../test-helper')
+const Promise = require('bluebird')
+
+const getNumberOfSubmittedClaimsForDateRange = require('../../../../app/services/data/get-number-of-submitted-claims-for-date-range')
+
+const REFERENCE = 'GETNUMC'
+const START_DATE = new Date('2016-12-08')
+const END_DATE = new Date('2016-12-08 23:59')
+
+describe('services/data/auto-approve-claim', function () {
+  before(function () {
+    var claim1 = testHelper.getClaimData(REFERENCE).Claim
+    claim1.DateSubmitted = new Date('2016-12-08 10:00')
+
+    var claim2 = testHelper.getClaimData(REFERENCE).Claim
+    claim2.ClaimId = claim1.ClaimId + 1
+    claim2.DateSubmitted = new Date('2016-12-08 23:00')
+
+    var claim3 = testHelper.getClaimData(REFERENCE).Claim
+    claim3.ClaimId = claim1.ClaimId + 2
+    claim3.DateSubmitted = new Date('2016-12-07 20:00')
+
+    return Promise.all([knex('IntSchema.Claim').insert(claim1),
+      knex('IntSchema.Claim').insert(claim2),
+      knex('IntSchema.Claim').insert(claim3)])
+  })
+
+  it('should return number of claims submitted', function () {
+    return getNumberOfSubmittedClaimsForDateRange(START_DATE, END_DATE)
+      .then(function (submittedClaimCount) {
+        expect(submittedClaimCount).to.equal(2)
+      })
+  })
```

Comment:
This test fails for me as I have other claims between these date ranges in my database.